src/plugins/saved_objects/public/finder/saved_object_finder.tsx

```diff
@@ -571,9 +607,20 @@ class SavedObjectFinderUi extends React.Component<
   }
 }
 
-const getSavedObjectFinder = (savedObject: SavedObjectsStart, uiSettings: IUiSettingsClient) => {
+const getSavedObjectFinder = (
+  savedObject: SavedObjectsStart,
+  uiSettings: IUiSettingsClient,
+  data: DataPublicPluginStart,
+  application: ApplicationStart
+) => {
   return (props: SavedObjectFinderProps) => (
-    <SavedObjectFinderUi {...props} savedObjects={savedObject} uiSettings={uiSettings} />
+    <SavedObjectFinderUi
+      {...props}
+      savedObjects={savedObject}
+      uiSettings={uiSettings}
+      data={data}
+      application={application}
+    />
```

Comment:
@kavilla If we want to do a quick fix for this breaking change, we could make the new parameters optional.
```
data?: DataPublicPluginStart,
application?: ApplicationStart
```

Reply:
Thanks @sejli I have taken your idea and made the new parameters as optional. Therefore if you don't pass in these params they should work in a similar way as before. 

To highlight the non breaking nature of changes I have reverted back the changes made to the state where we were using it without the new params to ensure it does n't break the functionality elsewhere.